diff --git a/font-patcher b/font-patcher
index f9c17689..c73337cd 100755
--- a/font-patcher
+++ b/font-patcher
@@ -817,11 +817,11 @@ class font_patcher:
             # also ensures the font is considered monospaced on Windows by setting the
             # same width for all character glyphs. This needs to be done for all glyphs,
             # even the ones that are empty and didn't go through the scaling operations.
-            self.set_glyph_width_mono(self.sourceFont[currentSourceFontGlyph])
 
             # Ensure after horizontal adjustments and centering that the glyph
             # does not overlap the bearings (edges)
             self.remove_glyph_neg_bearings(self.sourceFont[currentSourceFontGlyph])
+            self.set_glyph_width_mono(self.sourceFont[currentSourceFontGlyph])
 
         # end for
 
